platform/admin/ui/src/test/js/shared/utility.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    /*
+     Mocha Utility for Asynchronous Tests
+
+     If done is not called in asynchonous tests, mocha simply reports a timeout error.
+     However, we're more concerned with the assertion that threw the error than the
+     timeout.  This wrapper function ensures that done will be called with the appropriate
+     error (the assertion error), resulting in better test reports.
+     */
+    asyncAssertion: function (assertion, done) {
```

Comment:
I'd make `assertion` plural and, since this is a utility other people will use, add a note in the comment explaining what the parameter is and what it's used for.

Btw, doesn't JS have any kind of standard for comments, like Java does?